src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/SubNavigationTabs.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { Tabs, Tab } from "@madie/madie-design-system/dist/react";
+import {
+  CodesIcon,
+  AttributesIcon,
+  NegationRationaleIcon,
+} from "../../../../../../../icons";
+
+export interface NavTabProps {
+  activeTab: string;
```

Comment:
Do we need these for future ?

Reply:
Yes, this is required for sub navigation within the card